<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>800</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tahmeed">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>260</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 85, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/page1_back.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>startButton</zorder>
    </widget>
    <widget class="QWidget" name="Home">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 127);</string>
     </property>
     <widget class="QPushButton" name="Choice2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(167, 255, 255);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Choice1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>199</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/img/bakground.jpg</pixmap>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>Choice2</zorder>
     <zorder>Choice1</zorder>
    </widget>
    <widget class="QWidget" name="SearchPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 0, 0);</string>
     </property>
     <widget class="QLineEdit" name="search_in">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type a word</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Look it up</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 194, 163);
background-color: rgb(120, 154, 141);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/img/bakground.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>80</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 0, 77);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>search_in</zorder>
     <zorder>searchButton</zorder>
     <zorder>pushButton</zorder>
    </widget>
    <widget class="QWidget" name="AddPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 0);</string>
     </property>
     <widget class="QLineEdit" name="word_in">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Word</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pro_in">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Pronounciation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pos_in">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>40</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Parts of Speech</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mean_in">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>591</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Meaning</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="syn_in">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>591</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Similar Words</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="anto_in">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>591</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Opposite Words</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ex_in">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>591</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
padding-left: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Example</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>370</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 127, 127);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/img/bakground.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>80</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>word_in</zorder>
     <zorder>pro_in</zorder>
     <zorder>pos_in</zorder>
     <zorder>mean_in</zorder>
     <zorder>syn_in</zorder>
     <zorder>anto_in</zorder>
     <zorder>ex_in</zorder>
     <zorder>AddButton</zorder>
     <zorder>pushButton_2</zorder>
    </widget>
    <widget class="QWidget" name="ResultPage">
     <widget class="QLabel" name="show_word">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>231</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Word</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_pro">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 80, 129);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Pronounciation</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_pos">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 80, 129);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Parts of Speech</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_meaning">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>750</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(178, 172, 136);
background-color: rgb(215, 255, 178);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Meaning</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_syn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>750</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Synonym</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_anto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>750</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Antomyn</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="show_ex">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>750</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 248, 165);
padding-left: 10px;</string>
      </property>
      <property name="text">
       <string>Example</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/img/bakground.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : transparent; color : white;</string>
      </property>
      <property name="text">
       <string>Synonym</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : transparent; color : white;</string>
      </property>
      <property name="text">
       <string>Antonym</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : transparent; color : white;</string>
      </property>
      <property name="text">
       <string>Example</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>80</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</normaloff>:/img/dictionary-high-resolution-logo-black-transparent.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>show_word</zorder>
     <zorder>show_pro</zorder>
     <zorder>show_pos</zorder>
     <zorder>show_meaning</zorder>
     <zorder>show_syn</zorder>
     <zorder>show_anto</zorder>
     <zorder>show_ex</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
